src/HotChocolate/Core/src/Types.Scalars/LatitudeType.cs

```diff
@@ -0,0 +1,192 @@
+
+        /// <inheritdoc />
+        public override IValueNode ParseResult(object? resultValue)
+        {
+            return resultValue switch
+            {
+                null => NullValueNode.Default,
+
+                string s when Latitude.TryDeserialize(s, out var runtimeValue) =>
+                    ParseValue(runtimeValue),
+
+                int i => ParseValue(i),
+
+                double d => ParseValue(d),
+
+                _ => throw ThrowHelper.LatitudeType_ParseValue_IsInvalid(this)
+            };
+        }
+
+        /// <inheritdoc />
+        protected override double ParseLiteral(StringValueNode valueSyntax)
+        {
+            if (Latitude.TryDeserialize(valueSyntax.Value, out var runtimeValue))
+            {
+                return runtimeValue.Value;
+            }
+
+            throw ThrowHelper.LatitudeType_ParseLiteral_IsInvalid(this);
+        }
+
+        /// <inheritdoc />
+        protected override StringValueNode ParseValue(double runtimeValue)
+        {
+            if (Latitude.TrySerialize(runtimeValue, out var s))
+            {
+                return new StringValueNode(s);
+            }
+
+            throw ThrowHelper.LatitudeType_ParseLiteral_IsInvalid(this);
+        }
+
+        /// <inheritdoc />
+        public override bool TrySerialize(object? runtimeValue, out object? resultValue)
+        {
+            switch (runtimeValue)
+            {
+                case double d when Latitude.TrySerialize(d, out var serializedDouble):
+                    resultValue = serializedDouble;
+                    return true;
+
+                case int i when Latitude.TrySerialize(i, out var serializedInt):
+                    resultValue = serializedInt;
+                    return true;
+
+                default:
+                    resultValue = null;
+                    return false;
+            }
+        }
+
+        /// <inheritdoc />
+        public override bool TryDeserialize(object? resultValue, out object? runtimeValue)
+        {
+            if (resultValue is string s &&
+                Latitude.TryDeserialize(s, out var value))
+            {
+                runtimeValue = value;
+                return true;
+            }
+
+            runtimeValue = null;
+            return false;
+        }
+
+        private static class Latitude
+        {
+            private const double _min = -90.0;
+            private const double _max = 90.0;
+            private const int _maxPrecision = 8;
+
+            private const string _sexagesimalRegex =
+                "^([0-9]{1,3})°\\s*([0-9]{1,3}(?:\\.(?:[0-9]{1,}))?)['′]\\s*(([0-9]{1,3}" +
+                "(\\.([0-9]{1,}))?)[\"″]\\s*)?([NEOSW]?)$";
+
+            private static readonly Regex _validationPattern =
+                new(_sexagesimalRegex, RegexOptions.Compiled | RegexOptions.IgnoreCase);
+
+            public static bool IsValid(double value) => value is > _min and < _max;
```

Comment:
I have one question left then we will get this merged. 
I consolidate the conditions so that we only have it once in the code. The question is, shouldn't this be >= and <=max?

Reply:
I think it should be `< and >` only. Based on these similar implementations:

https://github.com/Urigo/graphql-scalars/blob/master/src/scalars/Latitude.ts#L27

https://github.com/manuelbieh/geolib/blob/e88635be6c8d75928307ec6df7f6e7c755296b15/src/isValidLatitude.ts#L9